bloom-instance/db/password.tf

```diff
@@ -30,6 +30,11 @@ resource "aws_secretsmanager_secret_version" "conn_string" {
   secret_id = aws_secretsmanager_secret.conn_string.id
   secret_string = jsonencode({
     "jdbc_uri" : local.conn_string,
```

Comment:
Let's keep these simple.

`uri` or `url` - since we aren't using jdbc
`pgpass` - is fine since it's temporary
`user` - since there's no need to limit ourselves to postgres
`password` - same
`host` - same
`port` - same
`db_name` - so it's clear what it refers to

Reply:
Will do. I can remove `pgpass`.

Reply:
Actually, it looks like `pgpass` isn't needed anymore in your latest buildspec changes.  If that's the case, just get rid of it since that's the only use case we had

Reply:
_done_